src/Registration/cReg/include/sirf/Reg/NiftyResample.h

```diff
@@ -42,6 +42,43 @@ class BSplineTransformation;
 
 namespace sirf {
 
+template<class dataType>
+class ComplexNiftiImageData
```

Comment:
do you consider as a temporary work-around? If so, I'd put it in a `detail` `namespace`. If not, I'd put it in its own file and add a bit of doxygen (probably useful to make its intent clear anyway)

Reply:
I don't consider it a temporary workaround, but I only see it being used as a wrapper in the resampler, so didn't really want to expose it to the rest of SIRF. What would you prefer?

Reply:
ok. if not to be exposed, let's put it in its own namespace. See [here](https://github.com/UCL/STIR/blob/e2f0bcb74af6bbf067785e07f58a8430faf3c64d/src/include/stir/IO/read_data_1d.h#L36) for an example

Reply:
done